corehq/ex-submodules/couchforms/models.py

```diff
@@ -224,6 +224,23 @@ def metadata(self):
 
         return None
 
+    @property
```

Comment:
i might have missed something from the other pr but any reason these are properties not saved directly on the doc

Reply:
The data already exists on the couch doc (without having to talk to a BLOB DB as in the case of SQL), so instead of saving again as properties, I have added interfaces on the couch doc (in the other PR)just to be consistent with SQL.

Reply:
makes a lot of sense. thanks